pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -650,7 +661,20 @@ func (dm *DockerManager) runContainer(
 			CgroupPermissions: device.Permissions,
 		}
 	}
-	binds := makeMountBindings(opts.Mounts)
+
+	// Propagation mode is set to:
+	// "" for non HostPath volumes
+	// "rshared" for HostPath volume mount in priviledged containers
+	// "rslave" for HostPath volume mount in non-priviledged containers
+	var propagation string
+	if dockerNewerThanV110 >= 0 {
+		if container.SecurityContext != nil && *container.SecurityContext.Privileged {
```

Comment:
It should be 
> container.SecurityContext != nil && container.SecurityContext.Privileged != nil && *container.SecurityContext.Privileged

`container.SecurityContext.Privileged` is a pointer and can be a nil.